tests/options/pugClassLocation/pug-class-location.test.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable quotes */
+import { format } from 'prettier';
+import { plugin } from './../../../src/index';
+
+describe('Options', () => {
+	describe('pugClassLocation', () => {
```

Comment:
These tests are a good start, but IMO they need to be more complex.

We should also test for
1. wrapped attributes
2. indented tags

So I think you should outsource the pug content to pug files like in most other tests to have better multiline.
Also in e.g. VSCode it's possible to make a file compare between formatted and unformatted, that help sometimes to work with tests.

Reply:
what is an example of wrapped attributes?

Reply:
`printWidth` ...

```pug
my-tag(
  my-attr="something"
)
```